src/domaines/classement/adapters/classement.presenter.impl.ts

```diff
@@ -0,0 +1,79 @@
+import { ClassementPresenter } from '@/domaines/classement/ports/classement.presenter';
+import { Classement, ClassementPourcentage } from '@/domaines/classement/recupererClassement.usecase';
+
+interface ClassementItemViewModel {
+  id: string;
```

Comment:
Pourquoi a-t-on besoin de l'id utilisateur ?
Si c'est pour le v-for, on peut simplement prendre l'index dans la liste.